app/src/main/java/br/edu/ifal/fiscalizaapp/composables/accordion/AccordionComposable.kt

```diff
@@ -0,0 +1,115 @@
+package br.edu.ifal.fiscalizaapp.composables.accordion
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.core.FastOutSlowInEasing
+import androidx.compose.animation.core.tween
+import androidx.compose.animation.expandVertically
+import androidx.compose.animation.shrinkVertically
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ExpandLess
+import androidx.compose.material.icons.filled.ExpandMore
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import br.edu.ifal.fiscalizaapp.composables.card.CardBase
+import br.edu.ifal.fiscalizaapp.ui.theme.FiscalizaTheme
+
+@Composable
+fun Accordion(
+    question: String,
+    answer: String,
```

Comment:
De início, crie:

```
data class AccordionItem(
    val question: String,
    val answer: String
)
```

```suggestion
    items: List<AccordionItem>
```

Reply:
Apenas dei este nome pois o componente Accordion já existe dentro desse arquivo. Caso tenha um nome melhor pode atribuir ao componente.